class ValidationError(Exception):
    pass


class ResponseError(Exception):
    pass
